app/views/messages/new.html.erb

```diff
@@ -1,60 +1,61 @@
-        Email
-        <br>
-        <br>
-        <div class="smallerfont">
-          support@indiearcade.com
+      <div class="col-centered col-xs-12 col-sm-3 contact-email">
+        <div class="contact-email">
+          <span class="glyphicon glyphicon-envelope" aria-hidden="true"></span>
+          Email
           <br>
-          corporate@indiearcade.com
+          <br>
+          <div class="smallerfont">
+            support@indiearcade.com
+            <br>
+            corporate@indiearcade.com
+          </div>
         </div>
       </div>
 
-      <div class="col-xs-3 col-centered">
-        <span class="glyphicon glyphicon-phone-alt" aria-hidden="true"></span>
-        Phone Support
-        <br>
-        <br>
-        <div class="smallerfont">778-XXX-XXXX</div>
+      <div class="col-centered col-xs-12 col-sm-3">
+        <div class="contact-details">
+          <span class="glyphicon glyphicon-phone-alt" aria-hidden="true"></span>
+          Phone Support
+          <br>
+          <br>
+          <div class="smallerfont">778-XXX-XXXX</div>
+        </div>
       </div>
 
-      <div class="col-xs-3 col-centered">
-        <span class="glyphicon glyphicon-dashboard" aria-hidden="true"></span>
-        Hours of Operation
-        <br>
-        <br>
-          <div class="smallerfont">
-          Monday-Friday: 8:30 AM to 9:00 PM (5:30 PM Vancouver)
+      <div class="col-centered col-xs-12 col-sm-3">
+        <div class="contact-details">
+          <span class="glyphicon glyphicon-dashboard" aria-hidden="true"></span>
+          Hours of Operation
           <br>
-          Closed: Saturday, Sunday and National Holidays
-          </div>
+          <br>
+            <div class="smallerfont">
+            Monday-Friday: 8:30 AM to 9:00 PM (5:30 PM Vancouver)
+            <br>
+            Closed: Saturday, Sunday and National Holidays
+            </div>
+        </div>
       </div>
     </div>
 
   <hr id= "hrcontact">
   <br>
     <div id="leavemess">Leave a message</div>
     <br>
-    <div id="marginend">
-      <div id="container-medium">
-        <div class="form-control1", class="form-group1">
-          <%= simple_form_for @message, url: message_home_path do |f| %>
-            <%= f.full_error :email %>
-            <%= f.full_error :content %>
-            <%= f.input :email, placeholder:"Email", label: false, :error => false %>
-            <%= f.input :content, as: :text, input_html: {rows: 10, cols: 10}, placeholder:"Message", label: false, :error => false %>
-        </div>
 
-        <div>
-          <%= f.button :submit%>
-        </div>
-        <% end %>
+      <div class="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8">
+        <%= simple_form_for @message, url: message_home_path do |f| %>
+          <%= f.full_error :email %>
+          <%= f.full_error :content %>
+          <%= f.input :email, placeholder:"Email", label: false, :error => false %>
```

Comment:
Please be consistent with using short-hard syntax for hashes so it should be `error: false`